resources/scponlyc_user.rb

```diff
@@ -0,0 +1,55 @@
+# To learn more about Custom Resources, see https://docs.chef.io/custom_resources.html
+
+resource_name :scponlyc_user
+
+default_action :create
+
+property :pub_dir, String, default: '/pub'
```

Comment:
I wonder if this property should be called ``write_dir`` instead to be more descriptive.